### PR TITLE
Get json from message

### PR DESCRIPTION
**Pull request by bhowmikp**

### CODE REVIEW

**Comment by wgranados** [comment]
Some issues that I got from a glance on https://github.com/wgma00/GmailBot/pull/1/commits/2c066c52e5e3e2542ce59c6240a7c6374ad064b0.
1) I believe you used `python` 2.7 instead of 3; indicated by your print statements
2) You may have installed some dependencies using pip like `http2lib` without updating the requirements file.

File: read.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
+import quickstart
+import base64
+import email
+from httplib2 import Http
```
Comment (wgranados) [inline read.py:4]: I receive an import error when running this with python

File: read.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    can be used to indicate the authenticated user.
+    msg_id: The ID of the Message required.
+
+  Returns:
+    A Message.
+  """
+  try:
+    message = service.users().messages().get(userId=user_id, id=msg_id).execute()
+
+    print 'Message snippet: %s' % message['snippet']
```
Comment (wgranados) [inline read.py:26]: python3 has print statements enclosed in brackets

Comment (bhowmikp) [reply read.py:26]: this is python2

**Comment by wgranados** [comment]
I'll convert it to python3 when I find the time.

**Pull request merged**